elements/lisk-transactions/src/4_multisignature_transaction.ts

```diff
@@ -65,6 +66,21 @@ export const multisignatureAssetFormatSchema = {
 	},
 };
 
+const setMemberAccounts = (
+	store: StateStore,
+	membersPublicKeys: ReadonlyArray<string>,
+) => {
+	membersPublicKeys.map(memberPublicKey => {
+		const address = getAddressFromPublicKey(memberPublicKey);
+		const memberAccount = { ...store.account.getOrDefault(address) };
+		memberAccount.publicKey = memberAccount.publicKey || memberPublicKey;
+		store.account.set(memberAccount.address, memberAccount);
+	});
+};
+
+const extractPublicKeysFromAsset = (assetPublicKeys: ReadonlyArray<string>) =>
```

Comment:
Maybe rename this function to extractPublicKeysFromVotes?

Reply:
this one is for Multi-signatures not votes